src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -155,6 +155,29 @@ public void insertWhenInvalidNumberOfParametersReturnsFalse() throws SQLExceptio
         assertFalse(databaseManager.insert(TEST_TABLE_NAME, tableRow));
     }
 
+    @Test
+    public void deleteWhenInvalidTableNameReturnsFalse() throws SQLException {
```

Comment:
throws SQLException is redundant

Reply:
You're right... and I'm confused that IDEA doesn't mark that anyhow

Reply:
It’s higlighted with the gray color. You can inspect such errors with analyze-inspect function

Reply:
Hm... in my version of IDEA it wasn't higlighted... Anyway thanks for your remark! )